src/System.Runtime/tests/System/DateTime.cs

```diff
@@ -2,7 +2,10 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Collections.Generic;
```

Comment:
could you please add the test case I mentioned to you which is listed in the issue 
https://github.com/dotnet/coreclr/issues/1322
Thx